packages/client/modules/userDashboard/components/OrgTeams/TeaserOrgTeamsRow.tsx

```diff
@@ -0,0 +1,54 @@
+import {Lock} from '@mui/icons-material'
+import React from 'react'
+import {useHistory} from 'react-router'
+import {Button} from '../../../../ui/Button/Button'
+import plural from '../../../../utils/plural'
+
+type Props = {
+  hiddenTeamCount: number
+  orgId: string
+}
+
+const TeaserOrgTeamsRow = (props: Props) => {
+  const {hiddenTeamCount, orgId} = props
+  const history = useHistory()
+
+  const handleSeePlansClick = () => {
+    history.push(`/me/organizations/${orgId}/billing`)
+  }
+
+  return (
+    <div className='block cursor-default select-none'>
+      <div className='flex items-center p-4'>
+        <div className='flex flex-1 flex-col py-1'>
+          <div className='flex items-center text-lg font-bold'>
+            {hiddenTeamCount} {plural(hiddenTeamCount, 'Hidden Team')}
+          </div>
+          <div className='flex items-center justify-between'>
+            <div className='text-gray-600'>
+              Parabol Enterprise includes our Org Admin role, which allows you to{' '}
+              <strong>see all teams</strong> in your organization.
+            </div>
+          </div>
+        </div>
+        <div className='flex items-center justify-center'>
+          <div className='flex h-12 w-8 flex-row items-center py-2'>
+            <Lock className='h-8 w-8 text-slate-600' />
+          </div>
+        </div>
+      </div>
+      <div className='px-4 pb-4'>
+        <Button
+          variant='primary'
+          shape='pill'
+          className='bg-pink-500 hover:bg-pink-600 px-6 py-2 text-base text-white'
```

Comment:
+1 I personally find it to be too much on the nose. I know we want to encourage people to upgrade, but having this upgrade button so big and highlighted compared to the team rows feels like an ad. I think because it's very distinct from the other team rows already that it wouldn't harm to make the text paler and maybe smaller. @ackernaut what do you think?

Reply:
I’m okay with subtle. We could yank the big button and put a link on Parabol Enterprise at the beginning of the sentence that takes them to the billing view. I think the text should stay the same to be legible in this case. Why make it hard to read why we are showing them the locked thing? I also think the lock icon could go a step down in scale so if it’s at 36 -> 24 or 24 -> 18.

Reply:
Here’s [an example in Figma](https://www.figma.com/design/JYFQNSeutaa0wW4PwpMND0/Team-and-Members-Management-(OrgAdmin)?node-id=274-578&node-type=frame&t=nkOw5eH5PZHmrp8c-11) cc @tianrunhe